.happy/terraform/modules/batch/main.tf

```diff
@@ -42,9 +42,9 @@ resource aws_batch_job_definition batch_job_def {
       "value": "${var.frontend_url}"
     }
   ],
-  "vcpus": 8,
+  "vcpus": 4,
```

Comment:
This is the VCPU use for the download job. It's single threaded and doesn't make use of multiple CPUs. Adjusting it down to save resources.